atomic_reactor/plugins/post_tag_from_config.py

```diff
@@ -102,12 +103,12 @@ def get_and_add_tags(self):
 
     def get_component_name(self):
         try:
-            name_label = str(get_preferred_label_key(self.labels, "name"))
-            name = self.labels[name_label]
+            parsed_labels = self.labels.copy()
```

Comment:
Curious what this is for.`Labels` doesn't modify the mapping it is given (should that be clearer in its documentation?).

Reply:
I was running into some weird errors when I ran that code, but they've resolved so I've removed the copy and simplified a bit.
Fixed.